### PR TITLE
Substitute vendor configuration files for white-label episode 2, the white labels strike back

### PR DESCRIPTION
**Pull request by thomasdiluccio**
… (#3360)" (#3370)"

This reverts commit 978dbfff6e988529966a949fbf9ca9f07322e125.

## Why

Closes #{ISSUE_NUMBER}

### CODE REVIEW

File: sites/platform/src/create-apps/multi-app/project-structure.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ They all have different configurations for how they serve the files. For more de
 
 {{< note >}}
 
-The `.platform` directory is located at the root, separate from your apps.
+The `{{% vendor/configdir %}}` directory is located at the root, separate from your apps.
```
Comment (chadwcarlson) [inline sites/platform/src/create-apps/multi-app/project-structure.md:58]: This subbed out correctly - I think its just the parameter I didn't set back while testing that's the issue here.

File: sites/platform/config/_default/params.yaml [hunk 2]
```diff
@@ -13,18 +13,25 @@ vendor:
     name: Platform.sh
     cli: platform
     env_prefix: PLATFORM
+    config:
+        version: 1
+        dir: .upsun
```
Comment (chadwcarlson) [inline sites/platform/config/_default/params.yaml:18]: .platform <-- this wasn't reset back

**Review by AnouckColson — APPROVED** [review]

File: sites/platform/src/create-apps/multi-app/project-structure.md [hunk 3]
```diff
@@ -308,12 +308,12 @@ For example, to change the source root of the `admin` app
...
 source:
     root: admin
 ```
 
 The `source.root` path is relative to the repository root.
 In this example, the `admin` app now treats the `admin` directory as its root when building.
 
-If `source.root` isn't specified, it defaults to the same directory as the `.platform/applications.yaml` (or `.platform.app.yaml`) file itself.
\ No newline at end of file
+If `source.root` isn't specified, it defaults to the same directory as the `{{< vendor/configfile "apps" >}}` (or `.{{< vendor/configfile "app" >}}`) file itself.
```
Comment (chadwcarlson) [inline sites/platform/src/create-apps/multi-app/project-structure.md:319]: Extra period here

File: sites/platform/src/create-apps/workers.md [hunk 4]
```diff
@@ -233,7 +233,7 @@ workers:
 ```
 
 There's a lot going on here, but it's all reasonably straightforward.
-The configuration in `.platform.app.yaml` takes a single Python 3.7 code base from your repository,
+The configuration in `.{{< vendor/configfile "app" >}}` takes a single Python 3.7 code base from your repository,
```
Comment (chadwcarlson) [inline sites/platform/src/create-apps/workers.md:236]: Another here

**Review by AnouckColson — DISMISSED** [review]

File: themes/psh-docs/layouts/shortcodes/guides/gatsby/headless-project.md [hunk 5]
```diff
@@ -19,4 +19,4 @@ When deploying a single app project [such as Gatsby](/guides/gatsby/deploy.html)
 the repository needs three configuration files that describe its infrastructure, described below in detail.
 For [multi-app projects](/create-apps/multi-app/_index.md),
 two of those files remain in the project root and are shared between Gatsby and {{ .Get "name" }}.
-Each app keeps its own app configuration file (`.platform.app.yaml`) in its subdirectory.
+Each app keeps its own app configuration file (`{{ partial "vendor/configfile" (dict "context" . "config" "services") }}`) in its subdirectory.
```
Comment (gilzow) [inline themes/psh-docs/layouts/shortcodes/guides/gatsby/headless-project.md:22]: on the previous version we were referencing `.platform.app.yaml` on the change we appear to be referencing the vendor's services configuration file?

**Review by gilzow — APPROVED** [review]
see other approval message

**Pull request merged**